RELEASE.md

```diff
@@ -21,23 +40,31 @@ This is a document to describe the release process for the Go builder. Since all
 Set up env variables:
 
 ```shell
-export GH_TOKEN=<PAT-token>
 export GITHUB_USERNAME="laurentsimon"
 # This is the existing slsa-verifier version used by the builder. (https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/actions/generate-builder/action.yml#L55)
-export VERIFIER_TAG="v1.3.2"
+export VERIFIER_TAG="v2.0.1"
 export VERIFIER_REPOSITORY="$GITHUB_USERNAME/slsa-verifier"
-# Release tag of the builder we want to release
-export BUILDER_TAG="v1.2.2"
+# Release tag of the builder we want to release. Release Candidates end with "-rc.#"
+export BUILDER_TAG="v1.5.0-rc.0"
 # Branch name for our test
 export BUILDER_REF="release/bad-verifier-$BUILDER_TAG"
 export BUILDER_REPOSITORY="$GITHUB_USERNAME/slsa-github-generator"
 export GH=/path/to/gh
+GH_TOKEN=$(${GH} auth token)
+export GH_TOKEN
 ```
 
-## Tagging
+## Release candidate
+
+The first step in creating a release is to create a release candidate. Release candidates exercise the release and testing process.
 
-Create a new tag for the official generator via [slsa-framework/slsa-github-generator/releases/new](https://github.com/slsa-framework/slsa-github-generator/releases/new).
-The tag _MUST_ be a "canonical" semantic version without metadata (`$BUILDER_TAG`). Shorter versions are not accepted by the builder's and verifier's code.
+If any tests fail for a release candidate you can address the issues and create a new release candidate after incrementing the release candidate number.
+
+### RC tagging
+
+Create a new tag for the Release Candidate via [slsa-framework/slsa-github-generator/releases/new](https://github.com/slsa-framework/slsa-github-generator/releases/new). The tag _MUST_ be a "canonical" [semantic version](https://semver.org/) without metadata (`$BUILDER_TAG`). Shorter versions are not accepted by the builder's and verifier's code.
+
+Release candidates should include a suffix indicating the release candidate number of the form `-rc.#` where `#` is a number starting from `0`.
 
 Set the title to `$BUILDER_TAG`.
```

Comment:
Add a note about what to put in the release description; link to CHANGELOG etc. Since the release may sit a while before the PR to update version references below is merged, include "this is an un-finalized pre-release" in the release description.